### ISSUE TITLE
the width of tf-runs-selector is not correct

### ISSUE THREAD

**Issue by leomao** [tensorflow/tensorboard#2358]
## Environment information (required)

``````
--- check: autoidentify
INFO: diagnose_tensorboard.py version c8952b173fd728c440f0b6bb64c1ecfc87a182b9

--- check: general
INFO: sys.version_info: sys.version_info(major=3, minor=7, micro=3, releaselevel='final', serial=0)
INFO: os.name: posix
INFO: os.uname(): posix.uname_result(sysname='Linux', nodename='desktop', release='5.1.9-arch1-1-ARCH', version='#1 SMP PREEMPT Tue Jun 11 16:18:09 UTC 2019', machine='x86_64')
INFO: sys.getwindowsversion(): N/A

--- check: package_management
INFO: has conda-meta: False
INFO: $VIRTUAL_ENV: None

--- check: installed_packages
INFO: installed: tensorboard==1.14.0
INFO: installed: tensorflow==1.14.0rc1
INFO: installed: tensorflow-estimator==1.14.0

--- check: tensorboard_python_version
INFO: tensorboard.version.VERSION: '1.14.0'

--- check: tensorflow_python_version
INFO: tensorflow.__version__: '1.14.0-rc1'
INFO: tensorflow.__git_version__: 'unknown'

--- check: tensorboard_binary_path
INFO: which tensorboard: b'/usr/bin/tensorboard\n'

--- check: readable_fqdn
INFO: socket.getfqdn(): 'localhost.localdomain'

--- check: stat_tensorboardinfo
INFO: directory: /tmp/.tensorboard-info
INFO: os.stat(...): os.stat_result(st_mode=16895, st_ino=54268, st_dev=43, st_nlink=2, st_uid=1000, st_gid=1000, st_size=60, st_atime=1560839895, st_mtime=1560839895, st_ctime=1560839895)
INFO: mode: 0o40777

--- check: source_trees_without_genfiles
INFO: tensorboard_roots (1): ['/usr/lib/python3.7/site-packages']; bad_roots (0): []

--- check: full_pip_freeze
INFO: pip freeze --all:
absl-py==0.7.1
alabaster==0.7.12
ansible==2.8.1
anytree==2.6.0
apipkg==1.5
appdirs==1.4.3
asn1crypto==0.24.0
astor==0.7.1
astroid==2.2.5
atari-py==0.1.15
atomicwrites==1.3.0
attrs==19.1.0
audioread==2.1.8
autopep8==1.4.4
Babel==2.7.0
backcall==0.1.0
bcrypt==3.1.6
bleach==3.1.0
blessings==1.7
blosc==1.8.1
boto==2.49.0
boto3==1.9.159
botocore==1.12.159
bpython==0.18
btrfsutil==1.1.1
bz2file==0.98
CacheControl==0.12.5
cached-property==1.5.1
cachetools==3.1.1
capstone==4.0.1
certifi==2019.3.9
cffi==1.12.3
chardet==3.0.4
chrome-gnome-shell==0.0.0
Click==7.0
cloudpickle==0.8.1
colorama==0.4.1
commonmark==0.8.0
cryptography==2.6.1
cupshelpers==1.0
cupy==6.0.0
curtsies==0.3.0
cycler==0.10.0
Cython==0.29.10
decorator==4.4.0
defusedxml==0.6.0
distlib==0.2.9
distro==1.4.0
docker==4.0.1
docker-compose==1.24.0
docker-pycreds==0.4.0
dockerpty==0.4.1
docopt==0.6.2
docutils==0.14
doit==0.31.1
editdistance==0.5.3
entrypoints==0.3
faiss==1.5.2
fastrlock==0.4
fasttext==0.8.22
feedparser==5.2.1
filebytes==0.9.21
Flask==1.0.3
future==0.17.1
gast==0.2.2
gevent==1.4.0
gmpy2==2.1.0b1
google-api-python-client==1.7.9
google-auth==1.6.3
google-auth-httplib2==0.0.3
greenlet==0.4.15
grpcio==1.21.4
gunicorn==19.9.0
gym==0.12.5
h5py==2.9.0
html5lib==1.0.1
httplib2==0.12.3
idna==2.8
imageio==2.5.0
imagesize==1.1.0
iniconfig==1.0.0
iotop==0.6
ipykernel==5.1.0
ipython==7.5.0
ipython-genutils==0.1.0
ipywidgets==7.4.2
isc==2.0
isort==4.3.4
itsdangerous==1.1.0
jedi==0.13.3
Jinja2==2.10.1
jmespath==0.9.4
joblib==0.13.2
jsonschema==3.0.1
jupyter-client==5.2.4
jupyter-console==6.0.0
jupyter-core==4.4.0
Keras-Applications==1.0.8
Keras-Preprocessing==1.1.0
kiwisolver==1.1.0
lazy-object-proxy==1.4.1
lensfun==0.3.2
librosa==0.6.3
lit==0.8.0.dev0
llvmlite==0.29.0
lockfile==0.12.2
louis==3.10.0
lxml==4.3.4
Markdown==3.1.1
MarkupSafe==1.1.1
matplotlib==3.1.0
mccabe==0.6.1
mistune==0.8.4
more-itertools==4.3.0
mpmath==1.1.0
msgpack==0.6.1
namcap==3.2.7
nbconvert==5.5.0.dev0
nbformat==4.4.0
networkx==2.2
nltk==3.4
nose==1.3.7
notebook==5.7.8
numba==0.44.0
numpy==1.16.4
oauth2client==4.1.3
olefile==0.46
packaging==19.0
pandocfilters==1.4.2
paramiko==2.4.2
parso==0.4.0
pep517==0.5.0
pew==1.2.0
pexpect==4.7.0
pickleshare==0.7.5
Pillow==6.0.0
pip==19.0.3
pipenv==2018.11.15.dev0
pluggy==0.11.0
ply==3.11
POT==0.5.1
progress==1.5
prometheus-client==0.6.0
prompt-toolkit==2.0.9
protobuf==3.7.0
psutil==5.6.3
ptyprocess==0.6.0
pwquality==1.4.0
py==1.8.0
pyalpm==0.8.5
pyasn1==0.4.5
pyasn1-modules==0.2.5
pybind11==2.2.4
pycairo==1.18.1
pycodestyle==2.5.0
pycparser==2.19
pycuda==2018.1.1
pycups==1.9.74
pycurl==7.43.0.2
pyelftools==0.25
pyglet==1.3.2
Pygments==2.4.2
PyGObject==3.32.1
pyinotify==0.9.6
pylint==2.3.1
PyNaCl==1.3.0
pynvim==0.3.2
pyOpenSSL==19.0.0
pyparsing==2.4.0
PyQt5==5.12.2
PyQt5-sip==4.19.17
pyrsistent==0.15.2
pyte==0.8.0
pytest==4.4.2
python-dateutil==2.8.0
python-jsonrpc-server==0.1.2
python-language-server==0.26.1
python-ptrace==0.9.3
pytoml==0.1.20
pytools==2019.1.1
pytz==2019.1
PyWavelets==1.0.3
pyxdg==0.26
PyYAML==5.1
pyzmq==18.0.1
ranger-fm==1.9.2
recommonmark==0.4.0
requests==2.22.0
resampy==0.2.1
retrying==1.3.3
ropper==1.11.11
rsa==4.0
ruamel.yaml==0.15.97
s3transfer==0.2.0
scikit-image==0.15.0
scikit-learn==0.21.2
scipy==1.3.0
Send2Trash==1.5.0
setuptools==41.0.1
simplegeneric==0.8.1
sip==4.19.17
six==1.12.0
smart-open==1.8.4
snowballstemmer==1.2.1
SoundFile==0.10.1
speedtest-cli==2.1.1
Sphinx==2.1.1
sphinx-rtd-theme==0.4.3
sphinxcontrib-applehelp==1.0.1
sphinxcontrib-devhelp==1.0.1
sphinxcontrib-htmlhelp==1.0.2
sphinxcontrib-jsmath==1.0.1
sphinxcontrib-qthelp==1.0.2
sphinxcontrib-serializinghtml==1.1.3
sphinxcontrib-websupport==1.1.0
SQLAlchemy==1.3.4
sympy==1.4
team==1.0
tensorboard==1.14.0
tensorboardX==1.6
tensorflow==1.14.0rc1
tensorflow-estimator==1.14.0
tensorflow-serving-api==1.14.0rc0
tensorflow-serving-api-gpu==1.14.0rc0
termcolor==1.1.0
terminado==0.8.1
testpath==0.4.2
texttable==1.6.1
thefuck==3.29
tldr==0.4.4
torch==1.1.0
torchvision==0.3.0
tornado==5.1.1
tqdm==4.28.1
traitlets==4.3.2
typed-ast==1.3.5
unicorn==1.0.1
uritemplate==3.0.0
urllib3==1.25.3
virtualenv==16.1.0
virtualenv-clone==0.5.3
wcwidth==0.1.7
webencodings==0.5.1
websocket-client==0.56.0
Werkzeug==0.15.2
wheel==0.33.4
Whoosh==2.7.4
widgetsnbextension==3.4.2
wrapt==1.11.1
xdot==1.0
``````

For browser-related issues, please additionally specify:

- Browser:
  - Firefox 67.0.2
  - Google Chrome 75.0.3770.90
- Screenshot:
![Screenshot from 2019-06-18 14-58-10](https://user-images.githubusercontent.com/1811303/59659807-8eba1000-91d9-11e9-9f5d-06f608f680f3.png)

## Issue description

After the commit 1aa9e53, the width of `tf-runs-selector` is not correct.  It seems that adding the style `width: calc(100% - 30px);` can make the width look better, but I'm not sure if this is the correct way to do it.
